gluon/custom_import.py

```diff
@@ -67,6 +67,13 @@ def custom_importer(name, globals=None, locals=None, fromlist=None, level=-1):
         import_tb = None
         try:
             try:
+                if ('gluon' not in name and not name.startswith('.') and
```

Comment:
Perhaps removing 'gluon.' from name and trying to import. If you just compare if exists gluon in name we will get more bugs... :(